website/views/layouts/layout.ejs

```diff
@@ -306,6 +306,35 @@
     (see https://developer.mozilla.org/en-US/docs/Web/API/Window/self) */ %>
     <script>delete window.self;</script>
 
+    <script>
+        var lastScrollTop = 0;
+        var header = document.querySelector('.header') || document.querySelector('.homepage-header')
+
+        window.addEventListener('scroll', windowedScrolled)
+
+        function windowedScrolled() {
+            var scrollTop = window.pageYOffset || document.documentElement.scrollTop
+            var fleetLogo = header.querySelector('img')
+            if(scrollTop == 0 && header.classList.contains('homepage-header')) {
+                header.classList.remove('sticky')
+                fleetLogo.src = '/images/logo-white-118x48@2x.png'
+                fleetLogo.style = 'width: 118px; height: 48px;'
+                return;
+            }
+            if (scrollTop > lastScrollTop && scrollTop > window.innerHeight) {
+                header.classList.add('translate-y-0')
+            } else {
+                if(header.classList.contains('homepage-header') && scrollTop > window.innerHeight * 0.2) {
```

Comment:
Can we use a set number of pixels instead of `window.innerHeight`?
When testing this on a large screen, I was seeing that the homepage header was reaching the content on the page before it has a background.
![image](https://user-images.githubusercontent.com/7445991/164814897-68019a83-9a6e-40c4-9a3a-88197c92cf59.png)

Reply:
I opted for using .clientHeight of the header just to make up for the mobile screen if the height is not the same as the desktop version which seems to work pretty well. Let me know what you think